app/controllers/artists_controller.rb

```diff
@@ -6,4 +6,7 @@ def index
   def show
     @artist = Artist.find(params[:id])
   end
+
+  def all_artists
```

Comment:
what's the reason for this method?

Reply:
It serves as an index for listing all artists. This page will later be modified to serve as this page: 
![image](https://user-images.githubusercontent.com/29110579/47463577-72291d80-d79b-11e8-91d9-859882ade8b4.png)